pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3125,6 +3130,80 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 						"[{\"interface\":\"net1\",\"name\":\"net1\",\"namespace\":\"default\"},{\"interface\":\"net2\",\"name\":\"net1\",\"namespace\":\"default\"}]")),
 			)
 		})
+
+		Context("interface status", func() {
+			const (
+				ifaceName   = "iface1"
+				networkName = "meganet"
+			)
+
+			DescribeTable("hotplugged interface", func(vmi *virtv1.VirtualMachineInstance, expectedIfaces ...virtv1.Interface) {
+				Expect(hotpluggedInterfaces(vmi)).To(ConsistOf(expectedIfaces))
```

Comment:
What is tested here? `hotpluggedInterfaces`? It is a method defined in the test.